shardingsphere-scaling/shardingsphere-scaling-core/src/main/java/org/apache/shardingsphere/scaling/core/job/task/incremental/LogPositionManager.java

```diff
@@ -15,25 +15,20 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.scaling.core.job.position;
```

Comment:
Position is also used in job preparer, so I think it should not be move to incremental task package.

Reply:
The LogPosition manager is the util to get current binlog(wal) position from database. I haven't found the suitable naming for it yet, just adjuest the package.
Now the task position metadata manager is different class, the manager will refactor soon.